Sources/MenuAnimator.swift

```diff
@@ -100,10 +117,24 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
     //
     var present: Bool = false
     var interactionInProgress: Bool = false
-    
-    private var transitionDuration: TimeInterval!
-    private var scaleDiff: CGFloat!
-    private var visibleContentWidth: CGFloat!
+
+    private var transitionDurationPortrait: TimeInterval!
```

Comment:
I think we should remove force-unwrapping here and change `var` to `let` declaration.